var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -129,26 +129,28 @@ def install_targets(self):
         ]
         return make_args + self.make_defs
 
-    @run_after('install')
-    def check_install(self):
-        spec = self.spec
```

Comment:
I looked closer, this is a regression. Someone needs to simply fix it. That was not a problem a day ago, but right now I see the issue as well.